micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -202,21 +204,25 @@
+    typeof Digit?.Utils?.locale?.getTransformedLocale === "function" &&
     !label?.includes(Digit.Utils.locale.getTransformedLocale(moduleName)) &&
     !label?.includes(Digit.Utils.locale.getTransformedLocale(masterName))
   ) {
     titleCode = Digit.Utils.locale.getTransformedLocale(`${moduleName}.${moduleName}_${label?.slice(0, -2)}`);
     additionalCode = label?.slice(-2);
   }
+
   return (
     <span>
       <div className={classNames} style={style}>
         <label htmlFor={id} className="control-label" id={"label_" + id}>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)